charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -1,10 +1,17 @@
 apiVersion: apps/v1
+{{- if .Values.repoServer.enableStatefulSet }}
+kind: StatefulSet
```

Comment:
I'm little bit worried about this implementation because repo server is stateless, not stateful and typically runs with HPA (we run 30+ replicas). This feature will break the HPA that targets Deployment + will create large amount of PVCs.